doc/shared/Install.rst

```diff
@@ -801,20 +801,34 @@ illustration only.
    .. note:: See additional information on building wtih
              SuperLU_DIST enabled in :numref:`Installation.CMake.ExternalLibraries`.
 
+.. cmakeoption:: SUPERLUDIST_DIR
+
+   Path to SuperLU_DIST installation.
+
+   Default: none
+
 .. cmakeoption:: SUPERLUDIST_INCLUDE_DIR
 
    Path to SuperLU_DIST header files (under a typical SuperLU_DIST
    install, this is typically the SuperLU_DIST ``SRC`` directory)
 
    Default: none
 
+   .. note::
+
+      This is an advanced option. This option is deprecated. Use :cmakeop:`SUPERLUDIST_INCLUDE_DIRS`.
```

Comment:
This says to use `SUPERLUDIST_INCLUDE_DIRS` instead, but this guide never mentions `SUPERLUDIST_INCLUDE_DIRS` to be read/referenced.  Should that be added?